src/components/Block/Block.test.tsx

```diff
@@ -1,24 +1,31 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import * as React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
+import { renderIntoDocument, findRenderedDOMComponentWithClass } from 'react-dom/test-utils';
 import Block, { BlockProps, GutterSize, TextColor, TextSize } from '.';
 
+const renderWithoutContext = (jsx: JSX.Element) => {
+  const renderedText = renderIntoDocument(jsx) as Block;
+  return findRenderedDOMComponentWithClass(renderedText, 'y-block');
+};
+
 describe('<Block />', () => {
-  let component: ShallowWrapper<BlockProps>;
+  let component: Element;
+  let componentWithContext: ShallowWrapper<BlockProps>;
 
   describe('with default options', () => {
     beforeEach(() => {
-      component = shallow(<Block>block content</Block>);
+      componentWithContext = shallow(<Block>block content</Block>);
```

Comment:
There are two cases that snapshot Block with the context provider, one where TextSize is set, and one without.  The rest of the time, I think we want to exclude the context from the snapshot, to avoid unnecessary churn.